roles/kubernetes/master/templates/kubeadm-config.v1alpha3.yaml.j2

```diff
@@ -47,7 +47,7 @@ apiServerCertSANs:
 {% for san in  apiserver_sans.split(' ') | unique %}
   - {{ san }}
 {% endfor %}
-certificatesDir: {{ kube_config_dir }}/ssl
+certificatesDir: {{ kube_cert_dir }}
```

Comment:
{{ kube_cert_dir }}  is define  /etc/kubernetes/pki ?
if not,please define to `/etc/kubernetes/pki`